transport-native-epoll/src/main/java/io/netty/channel/epoll/AbstractEpollStreamChannel.java

```diff
@@ -884,7 +884,6 @@ void epollInReady() {
 
                 if (close) {
                     shutdownInput();
-                    close = false;
```

Comment:
@Scottmitch actually I think this should stay as you already handled the close above